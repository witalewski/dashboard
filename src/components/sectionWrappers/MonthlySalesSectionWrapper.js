import React from 'react';
import styled from '@emotion/styled';
import { Section } from '../Section';

const StyledMonthlySalesSectionWrapper = styled.div`
  min-width: 400px;
  .section-content {
    display: flex;
    flex-direction: column;
    justify-content: center;
  }
`;

export const MonthlySalesSectionWrapper = ({ className, data }) => (
  <StyledMonthlySalesSectionWrapper className={className}>
    <Section title="Monthly Sales" className="monthly-sales">
      <div className="section-content">SECTION CONTENT</div>
    </Section>
  </StyledMonthlySalesSectionWrapper>
);
