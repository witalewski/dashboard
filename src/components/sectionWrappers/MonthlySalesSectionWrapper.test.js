import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import mockChartData from '../../fixtures/mockChartData.json';
import { MonthlySalesSectionWrapper } from './MonthlySalesSectionWrapper';

describe('MonthlySalesSectionWrapper', () => {
  it('matches snapshot with mock data', () => {
    const renderer = new ShallowRenderer();
    renderer.render(
      <MonthlySalesSectionWrapper data={mockChartData.monthlySales} />
    );
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
