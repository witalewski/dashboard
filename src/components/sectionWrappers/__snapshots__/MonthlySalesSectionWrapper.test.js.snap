// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MonthlySalesSectionWrapper matches snapshot with mock data 1`] = `
<ForwardRef(render)>
  <Section
    className="monthly-sales"
    title="Monthly Sales"
  >
    <div
      className="section-content"
    >
      SECTION CONTENT
    </div>
  </Section>
</ForwardRef(render)>
`;
